app/src/test/java/org/piwigo/ui/login/LoginViewModelTest.java

```diff
@@ -140,15 +140,15 @@
         viewModel.username.set(USERNAME);
         viewModel.password.set(PASSWORD);
 
-        viewModel.onLoginClick();
+        viewModel.testConnection(true, URL);
```

Comment:
the test case is called "call login if URL is valid"... Shouldn't we keep the "onLoginClick()" here?

Reply:
onLoginClick doesn't call login from itself anymore and delegates to a sub function..

So we are ending up with an error like this:

```
Wanted but not invoked:
userRepository.login(
    "https://piwigo.org/demo",
    "demo",
    "demo"
);
-> at org.piwigo.ui.login.LoginViewModelTest.callloginIfUrlValid(LoginViewModelTest.java:145)
Actually, there were zero interactions with this mock.
```